jde/modules/ROOT/pages/demo_poll_edi_events.adoc

```diff
@@ -1,44 +1,55 @@
-= *PortX JD Edwards Connector for Mule 4 – Demo – Poll EDI Events*
+= *PortX JD Edwards Connector for Mule 4 – Poll EDI Events*
 :keywords: add_keywords_separated_by_commas
 :imagesdir: images
 :toc: macro
 :toclevels: 2
 
+////
 toc::[]
+////
 
-== *Introduction*
+== Introduction
 
-The _PortX JDE Connector_ leverages the internal capabilities of Oracle’s _JD Edwards EnterpriseOne™_ solution to a broader extent, and uses the power of Oracle’s Java Dynamic Connector to provide interoperability with _JD Edwards EnterpriseOne™_ and external systems.
+The _PortX JDE Connector_ leverages the internal capabilities of Oracle’s _JD Edwards EnterpriseOne™_ solution and uses the power of Oracle’s _Java Dynamic Connector_ to provide interoperability with _JD Edwards EnterpriseOne™_ and external systems.
 
-After reading this guide, you should be able to poll for EDI events created on the EnterpriseOne server
+The following guide explains how to poll for EDI Events created on the EnterpriseOne server.
 
-At design time, the _PortX JDE Connector_ allows you to discover functions by name providing full access to all Business Function signatures. Given that discovery is a heavy duty process, the Connector manages a cache repository for all function’s metadata used at least once.
+The _PortX JDE Connector_ allows you to discover functions by name, providing full access to all Business Function signatures. The Connector manages a cache repository for all function’s metadata used at least once.
+
+The results of a function call are injected into the flow as a Map containing key-value pairs with the invocation’s output values. This approach allows you to construct complex flows and manage custom exception strategies.
 
-The results of a function call will be injected into the flow as a Map containing key-value pairs with the invocation’s output values. This approach gives the end user freedom of choice to construct complex flows and manage custom exception strategies.
 
 Furthermore, you can have all functions in the flow, processing within the same transaction, and commit or rollback all changes.
 
 === Prerequisites
 
-This document assumes that you are familiar with Oracle’s _JD Edwards EnterpriseOne™_ basics, Mule the https://docs.mulesoft.com/anypoint-studio/v/6/download-and-launch-anypoint-studio[Anypoint™ Studio] interface.
+A basic understanding of Oracle’s _JD Edwards EnterpriseOne™_, Mule, and https://docs.mulesoft.com/anypoint-studio/v/6/download-and-launch-anypoint-studio[Anypoint™ Studio] interface is assumed.
 
 === Requirements
```

Comment:
Items in a "Requirements section" - assuming there are more than one - are usually bulleted.

Reply:
I am testing out a "Getting Started" format.